Source/ToolCore/NETTools/NETProjectSystem.cpp

```diff
@@ -214,6 +217,7 @@ namespace ToolCore
 
             if (build)
             {
+                build->UnsubscribeFromEvent(E_NETBUILDRESULT); //unsubscribe in case it has been subscribed before
                 build->SubscribeToEvent(E_NETBUILDRESULT, ATOMIC_HANDLER(NETProjectSystem, HandleNETBuildResult));
```

Comment:
It is only possible to have one subscription to an event, so this is unnecessary as the previous one will be overwritten